### PR TITLE
Updated dwelling

### PR DESCRIPTION
**Pull request by Eanders1**

### CODE REVIEW

**Comment by brembs** [comment]
I just made sure the dwelling time analysis sits in the right place in the code. It was flagged as a conflict by GitHub. Now there are no more conflicts. You probably have to pull this again to your source.

**Comment by Eanders1** [comment]
example colorlearning
Example from color learning data

File: project.Rmd [hunk 1]
```diff
@@ -80,8 +80,8 @@ aspect_ratio = 4/nrow(sequence %>% na.omit())
 
 **Number of experimental groups:** `r paste(NofGroups)`
 ```{r singleflydata, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE, error=FALSE}
-exp_groups <- sapply(exp_groups, '[', seq(max(lengths(exp_groups)))) #make dataframe from list
```
Comment (brembs) [inline project.Rmd:83]: This is the one we need to keep!

**Comment by Eanders1** [comment]
It was to complicated to add all the missing pieces for project.rmd, so what I did instead is that I downloaded the latest version and copy/pasted the entire thing into my project.rmd on this branch.

**Pull request merged**